package.json

```diff
@@ -109,8 +109,8 @@
     "chunk-manifest-webpack2-plugin": "1.0.1",
     "core-js": "2.4.1",
     "fabric8-planner": "0.22.10",
-    "fabric8-runtime-console": "0.8.35",
-    "fabric8-stack-analysis-ui": "0.1.0",
+    "fabric8-runtime-console": "0.8.37",
```

Comment:
You need to use 0.8.36, since 0.8.37 includes the updated login-client that has only been merged into runtime. You don't even have it in stack-analysis yet.